ansible/roles/sufia/tasks/data-repo.yml

```diff
@@ -63,6 +63,31 @@
       dest: '{{ project_app_root }}/app/assets/images/carousel/'
     ignore_errors: True
 
+
+  - name: ensure font directory exists
+    file:
+      path: '{{ project_app_root }}/app/assets/fonts'
+      state: directory
+
+  - name: find old font files
+    find:
+      path: '{{ project_app_root }}/app/assets/fonts/'
+    register: fonts
+
+  - name: delete old font files
```

Comment:
If we are deleting contents from an `assets` directory, should we not also notify a handler to run the `tmp:clear` rake task to clear the cache?  I seemed to have problems when flipping between having and not having the fonts in `fonts.zip` installed during provisioning.  This appeared to be an application caching issue rather than a browser caching issue for me.